assets/scss/_theme.scss

```diff
@@ -125,3 +125,37 @@ body {
 .modal-open {
   overflow-y: auto !important;
 }
+
+.btn-group > {
+  .radio-btn-container {
+    flex-grow: 1;
+    .btn {
+      width: 100%;
+    }
+    &:not(:last-child) > {
+      .btn {
+        border-bottom-right-radius: 0 !important;
+        border-top-right-radius: 0 !important;
+      }
+    }
+    &:not(:first-child) > {
+      .btn {
+        border-bottom-left-radius: 0 !important;
+        border-top-left-radius: 0 !important;
+      }
+    }
+  }
+}
+
+.btn-group-lg > {
+  .radio-btn-container > {
+    .btn {
+      --cui-btn-padding-y: .5rem;
+      --cui-btn-padding-x: 1rem;
+      --cui-btn-font-size: 1.25rem;
+      --cui-btn-border-radius: var(--cui-border-radius-lg);
+    }
+  }
+}
```

Comment:
Was it related to the use of 'scoped' in an opening `<style lang="scss" scoped>` tag? This is sometimes helpful but as I add more styles to a component I end up dropping the 'scoped' because it doesn't capture elements that are not directly listed in the template (i.e. excludes elements that are parts of child/imported components), and I do my own scoping if necessary by giving the component a unique id or class that I can scope my styles to.